builtin/providers/aws/resource_aws_opsworks_permission.go

```diff
@@ -105,7 +105,7 @@ func resourceAwsOpsworksPermissionRead(d *schema.ResourceData, meta interface{})
 			found = true
 			d.SetId(id)
 			d.Set("id", id)
-			d.Set("allow_ssh", permission.AllowSudo)
+			d.Set("allow_ssh", permission.AllowSsh)
 			d.Set("allow_sodo", permission.AllowSudo)
```

Comment:
@janschumann I think the `allow_sodo` should be changed to allow_sudo to fix the typo